lib/mail/version_specific/ruby_1_9.rb

```diff
@@ -125,7 +125,11 @@ def Ruby19.q_value_decode(str)
     def Ruby19.param_decode(str, encoding)
       str = uri_parser.unescape(str)
       str = charset_encoder.encode(str, encoding) if encoding
-      str
+      decoded = str.encode(Encoding::UTF_8, :invalid => :replace, :replace => "")
```

Comment:
Let's use the [Unicode replacement char](https://en.wikipedia.org/wiki/Specials_(Unicode_block)#Replacement_character) rather than drop the character entirely.